addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -152,7 +152,7 @@ options.registry.carousel = options.Class.extend({
     onClone: function () {
         var id = 'myCarousel' + new Date().getTime();
         this.$target.attr('id', id);
-        this.$target.find('[data-slide]').attr('href', '#' + id);
+        this.$target.find('[data-slide]').attr('href', '#' + id).attr('data-target', '#' + id);
```

Comment:
In fact it should update data-target if it is currently using data-target and href if it is currently using href, not add both. Probably the same for the line with data-slide-to below.